playbooks/roles/certbot/tasks/main.yml

```diff
@@ -43,6 +43,14 @@
     path: /var/www/certbot
     state: directory
 
+- name: Create Certbot deploy hooks directory
+  file:
+    path: /etc/letsencrypt/renewal-hooks/deploy
+    state: directory
+
+- name: Copy the Certbot deploy hook to restart nginx on renewal
+  copy: src=restart-nginx.sh dest=/etc/letsencrypt/renewal-hooks/deploy/restart-nginx.sh mode=755
```

Comment:
Please use the multiline form here – we always prefer that style since we think it's more readable.